course_discovery/apps/course_metadata/management/commands/refresh_course_metadata.py

```diff
@@ -119,109 +114,113 @@ def handle(self, *args, **options):
                 for signal in (post_save, post_delete):
                     signal.disconnect(receiver=api_change_receiver, sender=model)
 
-            # For each partner defined...
-            partners = Partner.objects.all()
-
-            # If a specific partner was indicated, filter down the set
-            partner_code = options.get('partner_code')
-            if partner_code:
-                partners = partners.filter(short_code=partner_code)
-
-            if not partners:
-                raise CommandError('No partners available!')
-
             token_type = 'JWT'
-            for partner in partners:
-                logger.info('Retrieving access token for partner [{}]'.format(partner.short_code))
+            access_token = None
+            courses_api_url = None
+            partner = getattr(settings, 'PARTNER', None)
+            if not partner:
+                raise CommandError('No partner available!')
+
+            logger.info('Retrieving access token for all Organizations...')
 
+            for site in Site.objects.all():
                 try:
+                    access_token_url = 'http://{lms_domain}{end_point}/access_token'.format(
```

Comment:
- If we have several sites. we only use one of them. Because one site can fetch courses of all organizations.